src/stavenote.ts

```diff
@@ -614,6 +614,9 @@ export class StaveNote extends StemmableNote {
       const bbFlag = this.flag.getBoundingBox();
       boundingBox.mergeWith(bbFlag.move(flagX, flagY));
     }
+    for (let i = 0; i < this.modifiers.length; i++) {
+      boundingBox.mergeWith(this.modifiers[i].getBoundingBox());
```

Comment:
OK, so we are defining the stavenote's bounding box to include its modifiers now. Will this break any other things that relied on the stavenote's bounding box being defined differently?

Reply:
No, there are no other visual differences.